numpyro/distributions/continuous.py

```diff
@@ -468,6 +503,8 @@ class Pareto(TransformedDistribution):
     arg_constraints = {'alpha': constraints.positive, 'scale': constraints.positive}
     support = constraints.real
 
+    # FIXME: should we use `concentration` and `scale` for consistence with other distributions
+    # tensorflow uses `concentration` and `scale` though
```

Comment:
I think it is better if we can swap `alpha` and `scale` (which makes Pareto closer to scipy/tensorflow than pytorch). There we can also set default parameter `scale=1`.

Reply:
Seems reasonable to do.